system_test/common/aest_channels_SUITE.erl

```diff
@@ -105,10 +103,10 @@
 all() -> [
     test_simple_same_node_channel,
     test_simple_different_nodes_channel,
-    test_compat_with_initiator_node_using_fortuna_major_channel_version,
-    test_compat_with_responder_node_using_fortuna_major_channel_version,
-    test_compat_with_initiator_node_using_latest_stable_version,
-    test_compat_with_responder_node_using_latest_stable_version,
+    %% latest stable version expects not pinned block_hash
+    %% uncomment when updated
+    %%test_compat_with_initiator_node_using_latest_stable_version,
```

Comment:
The DB compatibility system tests will also fail.

Reply:
Actually no.

What we persist in the DB is the off-chain state `aesc_offchain_state:state()` and not the `#data{}` record. The changes introduced are in the `#data{}` only so the DB compatibility don't fail. What is more important is that old persisted DB will not be discarded as incorrect.

What happens with changes at hand is Alice can drop and later on she can reconnect using yet a different set of deltas provided.